portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,65 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Toni!</h1>");
+    //Initializing and adding in arrayList
+
+    private ArrayList<String> commentsArr;
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    commentsArr = new ArrayList<String>();
+    commentsArr.add(request.getParameter("fname"));
```

Comment:
would be better to store these parameter names in constant

Reply:
I am not sure because we are getting the user input for the request variable in order to allow them to set the number of comments

Reply:
I'm not following. the param key "fname" can be stored as a constant right?

Reply:
ohh i understand now, I was thinking you mean the httpservlet request and response variables lol